src/core_modules/capture-core/components/WidgetEnrollment/Actions/AddLocation/AddLocation.component.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+import { IconLocation16, MenuItem } from '@dhis2/ui';
+import React, { useState, useMemo } from 'react';
+import i18n from '@dhis2/d2-i18n';
+import { dataElementTypes } from '../../../../metaData';
+import { MapCoordinatesModal } from '../../../../components/MapCoordinates';
```

Comment:
Hey,
Does it make sense to move the `MapCoordinates` component inside the `WidgetEnrollment` folder? I suggest this in order to avoid adding global '`./../../../components/ `dependencies and try to keep it as self-contained as possible.
Thanks!